docs/content/docs/dev/datastream/fault-tolerance/serialization/types_serialization.md

```diff
@@ -167,6 +168,24 @@ input.keyBy(_.word)
 
 Flink supports all Java and Scala primitive types such as `Integer`, `String`, and `Double`.
 
+#### Common Collection Types
+
+Since Flink 2.0, Flink comes with dedicated serialization support for common Java collection types, which is more efficient than going
+through a general purpose serialization framework. Currently, only `Map`, `List`, `Set` and its super interface `Collection`
```

Comment:
it would be nice for the reader to see why it is more efficient as a high level.